src/app/app.component.html

```diff
@@ -23,19 +23,39 @@
               aria-controls="dropdown-basic"> <span class="flag-icon flag-icon-{{selected.code.toLowerCase()}}"></span> {{ selected.name }} <span class="caret"></span>
       </button>
       <ul id="dropdown-basic" *dropdownMenu class="dropdown-menu dropdown-menu-right scrollable-menu" role="menu" aria-labelledby="button-basic">
-        <li role="menuitem" *ngFor="let country of countryList" (click)="select(country)">
+        <li>
+          <div class=" px-4 py-2">
+            <input type="text" class="form-control" placeholder="Search your country..." [(ngModel)]="searchContent" (keyup)="searchCountry()" (click)="$event.stopPropagation();" />
```

Comment:
Can you update the placeholder to "Search a country" and add a magnifier on the left?